scanpipe/pipes/d2d.py

```diff
@@ -774,94 +777,240 @@ def _map_javascript_resource(
             resource.update(status=flag.MAPPED)
 
 
-def _map_about_file_resource(project, about_file_resource, to_resources):
-    about_file_location = str(about_file_resource.location_path)
-    package_data = resolve.resolve_about_package(about_file_location)
+class AboutFileIndexes(NamedTuple):
```

Comment:
What are the benefits of subclassing NamedTuple in place of a regular class or dataclass?

Also, a docstring for this class would be nice.

Reply:
yeah, dataclass would be better here indeed. Using that. Added docstring too, thanks!